composer.json

```diff
@@ -22,10 +22,11 @@
     "source": "https://github.com/bryanjhv/slim-session.git"
   },
   "require": {
+    "php": "^7.4 || ^8.0",
```

Comment:
Slim already enforces THIS.

Reply:
It does, but your library is also useful with other micro frameworks. I don't personally use it with slim, and thought it useful to include as a signal of what this library supports.

Reply:
I haven't thought about that.
Ok, this part is useful then.